NMDSdevopsServiceAdm/DataEngineering
# Req dependencies lockin

# Description
https://trello.com/c/D3iNLxVf/44-lock-in-dependency-versions-in-pipfile-to-prevent-unexpected-breakages
# How to test
run pipeline with no issues due to dependancies, post merging should be able to run data pipeline end to end 
# Developer checklist
